azavea/azavea-branding-guide
# Polish the downloads for the azavea logo

## Overview
Largely a cleanup of what was available to download on the Azavea branding guide.
- Indicate "print" or "screen" for CMYK and RGB directories respectively
- Add AI and EPS to CMYK and White
- Ensure that all are using the proper color profiles

### Demo
Full logo
<img width="882" alt="screen shot 2018-08-15 at 1 36 29 pm" src="https://user-images.githubusercontent.com/5672295/44163467-8c56c900-a091-11e8-9216-142d49afa776.png">
<img width="932" alt="screen shot 2018-08-15 at 1 36 37 pm" src="https://user-images.githubusercontent.com/5672295/44163468-8c56c900-a091-11e8-91df-28f02585a6ca.png">
<img width="932" alt="screen shot 2018-08-15 at 1 36 40 pm" src="https://user-images.githubusercontent.com/5672295/44163469-8c56c900-a091-11e8-9369-1673b07e9c2f.png">

Mark
<img width="932" alt="screen shot 2018-08-15 at 1 36 44 pm" src="https://user-images.githubusercontent.com/5672295/44163507-a4c6e380-a091-11e8-9db8-117020c2ba44.png">
<img width="932" alt="screen shot 2018-08-15 at 1 36 45 pm" src="https://user-images.githubusercontent.com/5672295/44163508-a4c6e380-a091-11e8-8fb2-653c00a58cce.png">
<img width="932" alt="screen shot 2018-08-15 at 1 36 48 pm" src="https://user-images.githubusercontent.com/5672295/44163509-a4c6e380-a091-11e8-8b47-00d487890bc8.png">

## Testing Instructions
1. `git pull`
2. Check out this branch
3. If your vagrant is already running, do `vagrant reload` > `vagrant ssh` > `./scripts/server`
3. Go to Design > Logos page
4. Click the first "Download assets" button
5. Open up the .zip file now on your computer
6. Ensure that notes from the overview are true (esp that AI and EPS are added to the CMYK folder)
7. Click the third "Download assets" button
8. Repeat 5-6

Closes #80
